net.go

```diff
@@ -87,6 +87,13 @@ const (
 	maxPushPullRequests    = 128 // Maximum number of concurrent push/pull requests
 )
 
+const (
+	nodeStateAlive   = "alive"
+	nodeStateDead    = "dead"
+	nodeStateLeft    = "left"
+	nodeStateSuspect = "suspect"
```

Comment:
You could add a method on `NodeStateType` instead:

```
func (t NodeStateType) metricsString() string {
		switch n.State {
		case StateAlive:
			return "alive"
		case StateDead:
			return "dead"
		case StateSuspect:
			return "suspect"
		case StateLeft:
			return "left"
		default:
			return ""
		}
}
```

Reply:
great call.

Reply:
fixed.